### ISSUE TITLE
Ability to set the IP for sites installed with Pilothouse

### ISSUE THREAD

**Issue by Jaace** [philipnewcomer/wpdocker#98]
I'm wondering if it's feasible to allow an individual to change the default 127.0.0.1 IP for Pilothouse sites to any IP, like, say `192.168.3.105`? Similarly to how VVV let's you modify the IPs used for your sites.

I noticed that this is hard-coded in run.js and sites.js: https://github.com/Pilothouse-App/Pilothouse/search?utf8=%E2%9C%93&q=127.0.0.1&type

**Comment by philipnewcomer** [comment]
It wouldn't be too difficult to make that user-configurable in PH. However, can you describe the use case for doing this?

**Comment by tylercherpak** [comment]
Correct me if I am wrong, would this allow a user to run multiple sets of containers?

If so, this could be useful when a project has its own development environment and you don't want to run docker-compose down/up to switch between project.

**Comment by Jaace** [comment]
@philipnewcomer @tylercherpak It turns out my specific use-case is solvable in a different way. My issue was that I was trying to get a Vagrant box to speak to my sites created with Pilothouse. Since they are 127.0.0.1, I was flabbergasted on how to get Vagrant to be able to reference them.

It turns out having NAT enabled on the Vagrant machine and referencing the site on the Vagrant `/etc/hosts` file solves this, using 10.0.0.2.

TL;DR - my specific issue is no longer but I was also just curious if it were possible (or even useful), specifically in the scenario Tyler is mentioning. Anyway, I will close this, but feel free to re-open if others are also in need.

**Comment by philipnewcomer** [comment]
>Correct me if I am wrong, would this allow a user to run multiple sets of containers?
> If so, this could be useful when a project has its own development environment and you don't want to run docker-compose down/up to switch between project.

At this point PH isn't designed to support multiple sets of containers, although I wouldn't rule that out for some point down the road. I do know that supporting multiple sets of containers would require much more in-depth changes to PH than just changing `127.0.0.1` to something else.
